packages/server/src/utils/createBackingStorageInstance.js

```diff
@@ -0,0 +1,26 @@
+const AWSFileHandler = require('./AWSFileHandler')
+const LocalFileSystemHandler = require('./LocalFileSystemHandler')
+
+/**
+ * Factory function returning an instance of the backing file storage to be used
+ */
+const createBackingStorageInstance = () => {
+
+    let storage = null
```

Comment:
Nothing major, but there's no need for this assignment, you could just return the newly created instances